appimage/run_appimage.sh

```diff
@@ -3,20 +3,12 @@
 PYTHON_CMD="$APPDIR"/opt/python3.7/bin/python3.7
 DOT_MONKEY="$HOME"/.monkey_island/
 
-configure_default_server() {
```

Comment:
how are these changes related to the creation of data dir?

Reply:
Data dir is created according to the server config, if passed. If not, it creates a default server config and then creates the data dir according to the server config. Since that is now being handled in the code, the run script doesn't need to handle it.